### ISSUE TITLE
pip install is not working

### ISSUE THREAD

**Issue by guillaumeeb** [dask/dask-jobqueue#36]
I was doing some python 2 tests and tried to pip install dask-jobqueue with the recently released version. That did not work:

```
$ pip install dask-jobqueue
Collecting dask-jobqueue
  Downloading https://files.pythonhosted.org/packages/01/0b/daa1abb1b3244e788acb94586b0b9a35953f62d6f2b1ccfb041a97a53c99/dask-jobqueue-0.1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-adBjL5/dask-jobqueue/setup.py", line 8, in <module>
        with open('requirements.txt') as f:
    IOError: [Errno 2] No such file or directory: 'requirements.txt'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-adBjL5/dask-jobqueue/
```

**Comment by lesteve** [comment]
https://github.com/dask/dask-jobqueue/commit/cf1ade08f17141e2e7477bafa8b36e1c9758f7ce should fix it.

**Comment by lesteve** [comment]
I guess we need to make a new release to close this issue. @jhamman would you have time to do it? It would probably take you less time than me since you did it very recently ...

Bonus points if you add the commands you use to do the release somewhere!

**Comment by lesteve** [comment]
We may as well fix the python2 issue #35 before doing a new release while we are at it!

**Comment by ChenxiXiao** [comment]
@jakirkham  @jhamman @lesteve I am seeing this error again when installing dask with 3.11.5 Python and Pip 23.2.1.pip23.3.1
